plugins/woocommerce/src/RestApi/Routes/V4/Orders/Schema/OrderFeeSchema.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * OrderFeeSchema class.
+ *
+ * @package WooCommerce\RestApi
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\RestApi\Routes\V4\Orders\Schema;
+
+defined( 'ABSPATH' ) || exit;
+
+use WC_Order_Item_Fee;
+use WP_REST_Request;
+
+/**
+ * OrderFeeSchema class.
+ */
+class OrderFeeSchema extends AbstractLineItemSchema {
+	/**
+	 * The schema item identifier.
+	 *
+	 * @var string
+	 */
+	const IDENTIFIER = 'order-fee';
+
+	/**
+	 * Return all properties for the item schema.
+	 *
+	 * Note that context determines under which context data should be visible. For example, edit would be the context
+	 * used when getting records with the intent of editing them. embed context allows the data to be visible when the
+	 * item is being embedded in another response.
+	 *
+	 * @return array
+	 */
+	public function get_item_schema_properties(): array {
+		$schema = array(
+			'id'         => array(
+				'description' => __( 'Item ID.', 'woocommerce' ),
+				'type'        => 'integer',
+				'context'     => self::VIEW_EDIT_EMBED_CONTEXT,
+				'readonly'    => true,
+			),
+			'name'       => array(
+				'description' => __( 'Fee name.', 'woocommerce' ),
+				'type'        => 'string',
+				'context'     => self::VIEW_EDIT_EMBED_CONTEXT,
+			),
+			'tax_class'  => array(
+				'description' => __( 'Tax class of fee.', 'woocommerce' ),
+				'type'        => 'string',
+				'context'     => self::VIEW_EDIT_EMBED_CONTEXT,
+			),
+			'tax_status' => array(
+				'description' => __( 'Tax status of fee.', 'woocommerce' ),
+				'type'        => 'string',
+				'context'     => self::VIEW_EDIT_EMBED_CONTEXT,
+				'enum'        => array( 'taxable', 'none' ),
+			),
+			'total'      => array(
+				'description' => __( 'Line total (after discounts).', 'woocommerce' ),
+				'type'        => 'string',
+				'context'     => self::VIEW_EDIT_EMBED_CONTEXT,
+			),
+			'total_tax'  => array(
+				'description' => __( 'Line total tax (after discounts).', 'woocommerce' ),
+				'type'        => 'string',
+				'context'     => self::VIEW_EDIT_EMBED_CONTEXT,
+				'readonly'    => true,
+			),
+			'taxes'      => $this->get_taxes_schema(),
+			'meta_data'  => $this->get_meta_data_schema(),
+		);
+
+		return $schema;
+	}
+
+	/**
+	 * Get an item response.
+	 *
+	 * @param WC_Order_Item_Fee $order_item Order item instance.
+	 * @param WP_REST_Request   $request Request object.
+	 * @param array             $include_fields Fields to include in the response.
+	 * @return array
+	 */
+	public function get_item_response( $order_item, WP_REST_Request $request, array $include_fields = array() ): array {
```

Comment:
`$include_fields` is not used?

Reply:
Needed for compatibility with the abstract schema method.